usr/share/rear/output/USB/Linux-i386/110_check_mounted_usb.sh

```diff
@@ -0,0 +1,27 @@
+# To be executed after output/default/100_mount_output_path.sh
+# Check whether the the USB directory is a moutpoint.
+# We need it to be a mountpoint, with a block device mounted there,
+# because later tasks (like installing bootloader) depend on it.
+# output/default/100_mount_output_path.sh should have mounted it,
+# but in case of e.g OUTPUT_URL=file://..., the user must mount
+# the USB manually.
+
+# Usually one would use a usb:// URL for OUTPUT=USB, but it does not
+# need to be enforced if the user mounts the URL properly.
+
+local usbdev
+
+if ! mountpoint $BUILD_DIR/outputfs ; then
+    if [ "$(url_scheme $OUTPUT_URL)" == file ] ; then
+        LogPrintError "For file:// OUTPUT_URL you need to mount the output disk at the path yourself"
+    fi
+    Error "OUTPUT_URL '$OUTPUT_URL' is not mounted"
+fi
+
+usbdev="$(findmnt -funo SOURCE --target $BUILD_DIR/outputfs)"
```

Comment:
It also errors out when
```
findmnt -funo SOURCE --target $BUILD_DIR/outputfs
```
fails for all arbitrary reasons
but it should only error out
when what is mounted at $BUILD_DIR/outputfs
is not a block device.